consul-l7-observability-kubernetes/step6.md

```diff
@@ -0,0 +1,26 @@
+
+In a third terminal, forward the Grafana UI to port 3000 with the following command.
+
+`export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=grafana" -o jsonpath="{.items[0].metadata.name}") && kubectl --namespace default port-forward $POD_NAME 3000 --address 0.0.0.0`{{execute T3}}
+
+You will receive the following output.
+
+```plaintext
+Forwarding from 127.0.0.1:3000 -> 3000
+Forwarding from [::1]:3000 -> 3000
+```
+
+Now, open a new tab and view port 3000. Login with
```

Comment:
Tab as in new terminal window or something else?

Reply:
Tab is the tab that I have configured Katacoda to provide, next to the Terminal tab.  I'll update this.